src/components/pages/users.js

```diff
@@ -69,6 +90,31 @@ class Users extends React.Component {
         render: this.renderRoleTags
       }
     ];
+
+    this.externalUserTableColumns = [
+      {
+        title: () => this.props.t('username'),
+        dataIndex: 'username',
+        key: 'username',
+        sorter: firstBy('username'),
+        render: this.renderUsername
+      }, {
+        title: () => this.props.t('email'),
```

Comment:
This column is always empty (for now, I guess it will have data once we have Shibolleth), is it worth keeping?

Reply:
Removed it, we can add it again after introducing Shibboleth (so maybe never)

Reply:
I actually hope we get that feature, because it's something that can help us strategically to get other Unis interested.